webportal/lambda/.chalice/config.json

```diff
@@ -50,7 +50,9 @@
         "TABLENAME_MANAGE_UPTIME_INDEX":"dev-scheduleuptime-username-index",
         "COGNITO_USER_POOL":"dev-sdc-dot-cognito-pool",
         "IDP_PROVIDER_ARNS":"arn:aws:cognito-idp:us-east-1:911061262852:userpool/us-east-1_RqSIpOZN3"
-      }
+      },
+      "subnet_ids": ["subnet-1d181f40", "subnet-b29f87d6"],
+      "security_group_ids": ["sg-98eae8ee"]
     },
```

Comment:
Holding off on prod until the CRB is approved

Reply:
Where/how are these used? Chalice uses them?

Reply:
https://aws.github.io/chalice/topics/configfile.html#subnet-ids

Reply:
If you are just going to wait for the CRB, can you make the prod changes too?